### ISSUE TITLE
Incompatible with OptiFabric

### ISSUE THREAD

**Issue by PoopingPants** [qouteall/ImmersivePortalsMod#472]
Optifabric is pretty important as it's the only way to get optifine with fabric
I checked if it wasn't the mod itself and it loaded (deleted it)
Checked it without optifabric and it loaded

**Comment by qouteall** [comment]
which version?

**Comment by Lyapsus** [comment]
In my identical situation there are Minecraft 1.16.4, ImmersivePortals 0.49, OptiFabric 1.8.10, Optifine 1.16.4_HD_U_G5 and Fabric API 0.26.1 respectively.
Game just crashes in about a second after getting in the world.

**Comment by qouteall** [comment]
@Lyapsus Did you disable render regions?

**Comment by Lyapsus** [comment]
Oh sorry forgot about it. But I dunno where to find this option, gonna try it right now.

**Comment by qouteall** [comment]
@Lyapsus Video options - performance (maybe)

**Comment by Lyapsus** [comment]
@qouteall sorry for so long.
Well, now it loads in the world at least:) Gonna poke around lil more on a week to test things further.
Thanks a lot:)

**Comment by Lyapsus** [comment]
@qouteall uh oh, the new crash.
I tried to take screenshot of open portal with 4x resolution from optifine settings and all I've got is 0 byte screenshot file and these logs.
[OptiFabric 4x screenshot crash.txt](https://github.com/qouteall/ImmersivePortalsMod/files/5584374/OptiFabric.4x.screenshot.crash.txt)
Gonna try other options like 1x screenshot size...
What do u think about compatibilty render mode in IP configs tho? It was turned on. What does it do?

edit: Well i can say that it could be vram problem or smth because without shaders 4x screenshots work perfectly fine.
With shaders portals are lagging as hell being in sight also.
